src/Compilers/VisualBasic/Portable/BoundTree/BoundObjectCreationExpression.vb

```diff
@@ -9,12 +9,21 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
     Friend Partial Class BoundObjectCreationExpression
 
+        Public Sub New(syntax As SyntaxNode, constructorOpt As MethodSymbol, arguments As ImmutableArray(Of BoundExpression), defaultArguments As BitVector, initializerOpt As BoundObjectInitializerExpressionBase, type As TypeSymbol, Optional hasErrors As Boolean = False)
```

Comment:
>defaultArguments As BitVector [](start = 126, length = 29)

It feels like we should simply add it to the constructor below, as a required parameter

Reply:
There are 20 callsites for the constructor below, so I'm going to add it as an optional parameter to avoid bloating this change even more.

---
In reply to: [190019411](https://github.com/dotnet/roslyn/pull/26781#discussion_r190019411) [](ancestors = 190019411)